packages/livekit-rtc/src/participant.ts

```diff
@@ -185,6 +190,60 @@ export class LocalParticipant extends Participant {
     });
   }
 
+  async sendChatMessage(
+    text: string,
+    destinationIdentities?: Array<string>,
+    senderIdentity?: string,
+  ) {
+    const req = new SendChatMessageRequest({
+      localParticipantHandle: this.ffi_handle.handle,
+      message: text,
+      destinationIdentities,
+      senderIdentity,
+    });
+
+    const res = FfiClient.instance.request<SendChatMessageResponse>({
+      message: { case: 'sendChatMessage', value: req },
+    });
+
+    const cb = await FfiClient.instance.waitFor<SendChatMessageCallback>((ev) => {
+      return ev.message.case == 'chatMessage' && ev.message.value.asyncId == res.asyncId;
+    });
+
+    if (cb.error) {
+      throw new Error(cb.error);
+    }
+    return cb.chatMessage!;
+  }
+
+  async editChatMessage(
+    editText: string,
+    originalMessage: ChatMessage,
+    destinationIdentities?: Array<string>,
```

Comment:
seems kinda weird that you can give different values here than to the original. this might also cause participants who join a call after an initial message is sent but before an edit to receive the edit and maybe have strange behavior? not sure if there's an easy way to solve though

Reply:
yeah, had the same feeling and opted to omit destinationIdentites on client-sdk-js altogether for now because of that